docs/lab_42.md.html

```diff
@@ -22,27 +22,15 @@
 | article        | a, the                                           |
 | preposition    | under, over, around, near, beside, ...           |
 
-## Part 2 - Reporting Phrases
+## Part 2: Making Sentences
 
-1. Write a custom `reporter` block called "noun phrase" that reports a noun phrase where each word is chosen randomly from the lists you created.  A noun phrase consists of an article, an adjective, and a noun in that order.
+1. Modify your script so that when you press the space bar, a random sentence is generated and a sprite says the resulting sentence.
 
-2. Write custom `reporter` blocks like "noun phrase" for each of the phrase types listed below.
+**BONUS**: Modify your script so that a noun phrase can either be the construction from part 1.2 or a single proper noun (e.g. a person's name).  Your script should randomly decide which version of a noun phrase to use.
```

Comment:
I think something might have gotten lost in this cleanup. A couple of things:

- Without the phrase type/construction table, it is a little ambiguous how the students are to create a sentence in 2.1
- This references part 1.2 which doesn't exist. I think this was mistaken even before this commit and it meant to reference the old 2.1
- This mentions a "noun phrase", but the step to write that reporter was removed

Happy to open a PR to fix if easiest. Thanks for all of your work with TEALS!